ArcGISRuntimeSDKQt_QMLSamples/Scenes/AnimateImagesWithImageOverlay/AnimateImagesWithImageOverlay.qml

```diff
@@ -44,14 +45,17 @@ Rectangle {
             if (fileNamesLength === 0)
                 return;
 
-            // create an image frame with the url to the image file and a extent
-            let imageFrame = ArcGISRuntimeEnvironment.createObject("ImageFrame", {
-                                                                       url: dataPath + "/" + imageFrameFolder.fileNames()[index],
-                                                                       extent: imageFrameExtent
-                                                                   });
+            // Create new ImageFrames from images in the folder
+            // Add them to a list so infinite ImageFrame objects are not created each loop
+            if (imageFrameList.length !== fileNamesLength) {
+                imageFrameList.push(ArcGISRuntimeEnvironment.createObject("ImageFrame", {
+                                                                              url: dataPath + "/" + imageFrameFolder.fileNames()[index],
+                                                                              extent: imageFrameExtent
+                                                                          }));
+            }
```

Comment:
Does this perform ok? No lag the first time? It might be better to put this in the Scene's loadStatusChanged signal handler once it is loaded. Then, once everything is ready to go, enable the button so users can begin the loop. This is just an idea - if everything works well, then this should work just fine too

Reply:
@ldanzinger There's no discernible difference between the initial loop and latter ones. The only difference between the previous version and the this PR is that we're pushing new objects to a list before referencing. I agree it's a bit of an odd workflow however.